libiocage/lib/Jail.py

```diff
@@ -289,6 +289,23 @@ def destroy(self, force=False):
 
         self.storage.delete_dataset_recursive(self.dataset)
 
+    def rename(self, new_name: str):
+        """
+        Change the name of a jail
+        """
+
+        self.require_jail_existing()
+        self.require_jail_stopped()
+
+        current_id = self.config["id"]
+        dataset = self.dataset
+        self.config["name"] = new_name  # validates new_name
```

Comment:
huh, how??

Reply:
It should throw a KeyError I believe, but if so it's missing a try block.